### ISSUE TITLE
Covariance of bins not measured should be infinity (not zero)

### ISSUE THREAD

**Issue by andreufont** [igmhub/picca#778]
I was looking at how Picca does the sub-sampling, and it looks like it assumes a zero variance for bins not measured.

https://github.com/igmhub/picca/blob/02496df79e088332344e2f9322b681d71117ecbf/py/picca/utils.py#L56

I'm not sure this is used anywhere relevant, but I thought I'd flag it here just in case.

**Comment by ngbusca** [comment]
I agree, but bins that aren't measured aren't used in the fit. Isn't it equivalent? In other words, isn't the inverse covariance of the *measured* bins equal to the inverse of the covariance where you delete rows and columns of unmeasured separations?

**Comment by andreufont** [comment]
Hi Nico, welcome back to Picca!

Yes, what you are saying is correct, and I believe this "issue" is not really an issue in our standard analysis. I just wrote it here as a warning, not as something that needs to be fixed soon.
